### PR TITLE
Make data dirs for all pods when syncing

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

**Comment by thockin** [comment]
Don't commit this until cleanup has been added

**Comment by thockin** [comment]
I think this is ready to review, assuming it goes green (passed locally :)

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -234,6 +242,31 @@ func dirExists(path string) bool {
 	return s.IsDir()
 }
 
+func (kl *Kubelet) setupDataDirsOrDie() {
```
Comment (smarterclayton) [inline pkg/kubelet/kubelet.go]: Why does this has to be a die?  Why not just return error from here?

Comment (thockin) [inline pkg/kubelet/kubelet.go]: Good point, done.

**Comment by thockin** [comment]
New push is up.  Would love to get this in today to hopefully finish volume plugins.

**Comment by dchen1107** [comment]
Seen this failure:
_output/local/go/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/runonce_test.go:72: kb.setupDataDirsOrDie undefined (type *Kubelet has no field or method setupDataDirsOrDie)

**Comment by thockin** [comment]
Fixed and pushed

**Comment by dchen1107** [comment]
LGTM.

**Pull request merged**